cardano-api/src/Cardano/Api/IPC.hs

```diff
@@ -4,6 +4,7 @@
 {-# LANGUAGE GADTs #-}
 {-# LANGUAGE NamedFieldPuns #-}
 {-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeApplications #-}
```

Comment:
I cannot see type applications in the diff :thinking:

Reply:
Removed.